### PR TITLE
Blockchain-based charity foundation platform

### PR DESCRIPTION
**Pull request by zemtsov**
Signed-off-by: Alexander Zemtsov <a.zemtsov@gmail.com>

## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

**Comment by C0rWin** [comment]
I think this is an exciting project. It allows solving trust issues by bringing blockchain to organize donations with complete transparency, allowing charity and non-profit organizations to collect funds while sharing details. Moreover, this is also an interesting combination where the project utilizes Hyperledger Fabric where architecture shares the same stack across frontend, backend and the chaincode implementation with NodeJS. 

Since the current repo doesn't have a commit's signoff, I guess the intent would be to start with the new repo and apply DCO.

**Review by tkuhrt — APPROVED** [review]
Approving this. Please let us know what approach you will take for DCO sign off.

**Comment by OlgaIlyina** [comment]
> 

@tkuhrt sounds great, will be happy to present it next week

File: labs/karma-charity-platform.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Initial Committers
+- https://github.com/ManhattanDoctor
+- https://github.com/zemtsov
+
+# Sponsor
+- [Artem Barger](https://github.com/C0rWin) - Member of the Hyperledger Technical Steering Committee.
+
+# Pre-existing repository
+https://github.com/ANO-ID-AS-Center/karma
```
[the remark this answers is not in the document]
Comment (C0rWin) [reply labs/karma-charity-platform.md:29]: I think the inly option would be to 

> Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs GitHub organization.

Cause to amend all commits is too tedious.

**Review by vipinsun — APPROVED** [review]
Existing code to be brought in manually. Looks good so far. Assuming @C0rWin is sponsoring from his comments.

**Comment by tkuhrt** [comment]
Created https://github.com/hyperledger-labs/karma-charity-platform

**Comment by tkuhrt** [comment]
@OlgaIlyina : Just wanted to check to see if you still wanted to present at this week's TSC call (June 16, 2022 @ 7:00 AM US/Pacific). We can provide 15 minutes total.

**Comment by OlgaIlyina** [comment]
@tkuhrt yes, will be glad to present

**Pull request merged**